src/install-peerdeps.js

```diff
@@ -98,6 +100,27 @@ function spawnInstall(command, args) {
   });
 }
 
+/**
+ * Read package file file from disk.
+ * Returns undefined if no package file exists.
+ */
+function readPackageFile() {
+  return new Promise((resolve, reject) => {
+    const packageFilePath = path.join(process.cwd(), "package.json");
+    if (fs.existsSync(packageFilePath)) {
+      fs.readFile(packageFilePath, "utf8", (err, packageFileJson) => {
+        if (err) {
+          reject(err);
+        } else {
+          resolve(JSON.parse(packageFileJson));
+        }
+      });
+    } else {
+      resolve();
```

Comment:
My understanding of what occurs in this line is that if the package file does not exist, `readPackageFile` resolves to `undefined`. Is this the desired behavior?

Reply:
Of course, just checking as I wasn't sure if you wanted to reject with an error or resolve with `false` instead -- regardless I see you're checking for truthiness below this line anyways, so `undefined` works for me.